pypeit/core/coadd.py

```diff
@@ -3073,6 +3074,11 @@ def run(self):
         self.waves, self.fluxes, self.ivars, self.masks, self.header = self.load()
         # Coadd the data
         self.wave_coadd, self.flux_coadd, self.ivar_coadd, self.mask_coadd = self.coadd()
+        # Scale to a filter magnitude?
```

Comment:
Should this be a command line argument rather than a parset parameter. The problem is we have only one parser for the .coadd files, and we tend to lump everything into parsets, but I'd much rather separate defaults and parameters which are in the parset from run time parameters which should be passed via the command line or passed via the .coadd, .flux etc. files.

Reply:
Well, it would require 4 command line arguments.
And it is nice having them recorded in the coadd file.